common/app/views/fragments/contentMeta.scala.html

```diff
@@ -56,7 +56,7 @@
     @if(item.tags.contributors.length == 1 && item.tags.contributors.headOption.exists(_.properties.twitterHandle.nonEmpty)) { content__meta-container--twitter}
     @if(item.tags.contributors.length == 1 && item.tags.contributors.headOption.exists(_.properties.emailAddress.nonEmpty)) { content__meta-container--email}">
 
-    @if(showBadge) {
+    @if(showBadge && (NewCommercialContent.isSwitchedOff || (NewCommercialContent.isSwitchedOn && (item.tags.isVideo || item.tags.isAudio || (item.tags.isArticle && !item.commercial.isAdvertisementFeature))))) {
```

Comment:
Good catch! If the switch is off, then we fallback to the original test (ie `showBadge`). If the switch is on, the badge should only be shown on video/audio pages as well as on articles which are **not** paid for ones (in the latter case, the badge will appear somewhere else)